Source/WebCore/Modules/webaudio/AudioContext.cpp

```diff
@@ -507,14 +643,17 @@ bool AudioContext::shouldOverrideBackgroundPlaybackRestriction(PlatformMediaSess
     }
 #endif
 
-    return !destination().isConnected();
+    RefPtr document = this->document();
+    return hasPlayBackAudioSession(RefPtr { this->document() }.get());
```

Comment:
that time convoluted; what's `document` for here ?

Reply:
Oh, yes it should be hasPlayBackAudioSession(document.get()).

Reply:
Fixed